packages/synapse-react-client/src/components/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -149,21 +142,18 @@ export const applyCombinedChangesToRangeColumn = (
 /**
  * Determines which facet filters should be shown after loading a new bundle. The shown facets will be the first
  * three available facets, plus any other facets that have a filter applied.
- * @param facets
+ * @param facetColumns
  * @param selectedFacets
  * @returns the columnNames of the facets that should be shown.
  */
 export function getDefaultShownFacetFilters(
-  facets: FacetColumnResult[],
+  facetColumns: string[],
```

Comment:
Since one column may now have multiple facets, the show/hide facet control functionality now operates on columns rather than facets